src/chart/generateCategoricalChart.tsx

```diff
@@ -232,7 +232,7 @@ const getTooltipContent = (
       return result;
     }
 
-    const { data } = child.props;
+    const { data = chartData } = child.props;
```

Comment:
could we add a comment here explaining why this needs defaulted? I feel this is something we'll loose context on later and break accidentally

Reply:
@ckifer Added comment. Thanks!